### DISCUSSION TITLE
Execution of c/c++ files

### DISCUSSION THREAD

**Discussion by SLQAVEVVEVEVE** [thebaselab/codeapp#740] [Q&A]
How to run cpp file without clang++ (maybe with 9.2.0 or smth like that)

**Comment by bummoblizard** [comment]
Do you mean gcc? The recent update disabled it. Let me know if you want it back.

**Comment by SLQAVEVVEVEVE** [comment]
Yeah, like clang is not really good for my purposes Отправлено с iPad26 янв. 2023 г., в 21:31, Chung Shing Hin ***@***.***> написал(а):﻿
Do you mean gcc? The recent update disabled it. Let me know if you want it back.
